packages/react-components/src/TextArea.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright 2017-2020 @polkadot/react-components authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import styled, { css } from 'styled-components';
+
+interface Props {
+  withError?: boolean;
+}
+
+const TextInput = css(({ withError }: Props) => `
```

Comment:
Def. prefer none of these components. Just standard React components instead of the style-component shorthands. 

Yes, this is the way _they_ recommend, but sadly something that I find non-maintainable, nor readable.